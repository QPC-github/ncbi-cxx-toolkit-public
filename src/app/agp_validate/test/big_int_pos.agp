# COMMENT: Generated by ALLMAPS v0.5.3 (2018-06-08)								
# COMMAND: python -m jcvi.assembly.allmaps path --cpus 48 Genotypes_Allnewsplit_Xw3_RF_WGw2_AGOUTI_FULLw2_RF_genemodw1.tabs.bed /box85T/SALAMANDER/Assemblies/Clean_fasta_assemblies/WG/Tamapka2018/AmexG_v3.0.0.NEW.SOFTMASKED.fa -w weights.txt --ngen 0 --npop 1								
# FIELDS: object, object_beg, object_end, part_number, component_type, component_id/gap_length, component_beg/gap_type, component_end/linkage, orientation/linkage_evidence
chr1	1	2521255	1	W	PGSH01056636.1	1	2521255	+
chr1	2521256	2521355	2	U	100	scaffold	yes	map
chr1	2521356	3938757	3	W	PGSH01025211.1	1	1417402	+
chr1	3938758	3938857	4	U	100	scaffold	yes	map
chr1	3938858	5455182	5	W	PGSH01099427.1	1	1516325	+
chr1	5455183	5455282	6	U	100	scaffold	yes	map
chr1	4294967300	4294967306	7	W	PGSH01018958.1	1	6	+
chr1	2147385906	-2146453863	4221	W	PGSH01000058.1	1	1127528	+
chr1	-2146453862	-2146453763	4222	U	100	scaffold	yes	map
chr1	4294967290	4294967294	7	W	PGSH01018960.1	1	5	+
