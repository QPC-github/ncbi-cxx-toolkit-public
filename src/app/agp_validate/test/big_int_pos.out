
5:chr1	2521256	2521355	2	U	100	scaffold	yes	map
	NOTE: assuming AGP version 2 since linkage evidence (column 9) is NOT empty

10:chr1	4294967300	4294967306	7	W	PGSH01018958.1	1	6	+
	ERROR, line skipped: object_beg (column 2) must not exceed 4294967294
	ERROR, line skipped: object_end (column 3) must not exceed 4294967294

11:chr1	2147385906	-2146453863	4221	W	PGSH01000058.1	1	1127528	+
	ERROR, line skipped: object_end (column 3) must be a positive integer

12:chr1	-2146453862	-2146453763	4222	U	100	scaffold	yes	map
	ERROR, line skipped: object_beg (column 2) must be a positive integer
	ERROR, line skipped: object_end (column 3) must be a positive integer

5 errors, no warnings, 1 note.
  Count Code  Description
      3  e06  X must be a positive integer
      2  e07  X must not exceed 4294967294
      1  w58  assuming AGP version X

NOTE: 3 invalid lines were skipped (not subjected to all the checks, not included in most of the counts below).


Objects                : 1
- with single component: 0

Scaffolds              : 1
- with single component: 0

Components                   : 4 (W)
  orientation +              : 4
  orientation -              : 0
  orientation ? (formerly 0) : 0
  orientation na             : 0

Gaps                   : 3 (U)
- do not break scaffold: 3
  scaffold, linkage yes: 3
- break it, linkage no : 0

Linkage evidence:
  map: 3

Object names:  1
  chr1      :  1

Component names             :  4
  PGSH[01018960..01099427].1:  4

#Comment line count    : 3
